apps/video/test/marionette/lib/video.js

```diff
@@ -263,6 +285,14 @@ Video.prototype = {
   },
 
   /**
+   * Click option button from player view and wait for button selection
+   */
+  tapOption: function() {
+    this.optionsButton.click();
+    this._createOptionsView().waitForOptionsViewDisplayed();
```

Comment:
I like how you have introduced 'waitForOptionsViewDisplayed' in VideoOptionsView. Using that in the test code before invoking the video info view will prevent any timing issues. Since the test is calling 'waitForOptionsViewDisplayed', 'tapOption' doesn't need to call it. And then '_createOptionsView' is not needed :)